portafly/.eslintrc.js

```diff
@@ -0,0 +1,33 @@
+module.exports = {
+  root: true,
+  parser: '@typescript-eslint/parser',
+  plugins: [
+    'jest'
+  ],
+  extends: [
+    'airbnb-typescript',
+    // 'plugin:jest/recommended'
```

Comment:
Is this plugin needed?

Reply:
Oops, I might have left some comments there